.gitignore

```diff
@@ -40,6 +40,7 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
+config/
```

Comment:
I am using my own database from env but it's not working that's why I put it into git ignore.
Whenever I will change this files in local machine it will ignore change to push from machine
for this directory and files

Reply:
it's only for prevent from changes in local machine

Reply:
I changed as before please check